packages/venia-concept/src/ServiceWorker/Utilities/htmlHandler.js

```diff
@@ -0,0 +1,17 @@
+const cloneRequestWithDiffURL = (request, url) =>
+    url
+        ? new Request(url, {
+              method: request.method,
+              headers: request.headers
+          })
+        : request;
+
+export const cacheHTMLPlugin = {
+    cacheKeyWillBeUsed: async () => {
+        return '/';
+    },
+    requestWillFetch: async ({ request }) => {
+        const newRequest = cloneRequestWithDiffURL(request, '/');
```

Comment:
Just to check my understanding, if an "html route" comes in (`/` or ends in `.html`), we treat it as if it was `/` and just return the cache results from that. Or more specifically, issue a new request and let workbox handle returning the correct value.

Do I have that right?

Reply:
Nailed it.